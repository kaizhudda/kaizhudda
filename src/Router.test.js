import React from 'react';
import { shallow } from 'enzyme';
import Router from './Router';

let wrapped = shallow(<Router />); 

describe('Router', () => {
  it('should render the Router Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should contain 1 Route', () => {
    expect(wrapped.find('Route').length).toBe(1);
  });
});
