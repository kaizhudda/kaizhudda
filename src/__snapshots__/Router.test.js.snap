// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router should render the Router Component correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Switch>
`;
